lib/vtls/schannel.c

```diff
@@ -1690,6 +1690,8 @@ enum schannel_renegotiate_caller_t {
   SCH_RENEG_CALLER_IS_SEND
 };
 
+#define MAX_REGEG_BLOCK_TIME (7 * 1000) /* 7 seconds in milliseconds */
```

Comment:
REGEG/RENEG but IMO this is really a one line change where you could have set max_renegotiate_ms to 7 * 1000